test/e2e/packetcapture_test.go

```diff
@@ -586,6 +606,24 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 	if !packetCaptureStatusEqual(pc.Status, tc.expectedStatus) {
 		t.Errorf("CR status not match, actual: %+v, expected: %+v", pc.Status, tc.expectedStatus)
 	}
+
+	if tc.expectedStatus.NumberCaptured == 0 {
+		return
+	}
+	// verify packets.
+	antreaPodName, err := data.getAntreaPodOnNode(nodeName(0))
+	require.NoError(t, err)
+	fileName := fmt.Sprintf("%s.pcapng", tc.pc.Name)
+	dstFileName := filepath.Join(os.TempDir(), fileName)
+	packetFile := filepath.Join("/tmp", "antrea", "packetcapture", "packets", fileName)
+	require.NoError(t, data.copyPodFiles(antreaPodName, "antrea-agent", "kube-system", packetFile, os.TempDir()))
+	file, err := os.Open(dstFileName)
+	require.NoError(t, err)
+	defer func() {
+		defer file.Close()
+		defer os.Remove(dstFileName)
+	}()
```

Comment:
what is the point of having 2 defer statements in a deferred function?

Couldn't the code be like this:
```go
require.NoError(t, data.copyPodFiles(antreaPodName, "antrea-agent", "kube-system", packetFile, os.TempDir()))
defer os.Remove(dstFileName)
file, err := os.Open(dstFileName)
require.NoError(t, err)
defer file.Close()
```

this would close the file first, then remove it (reversed order of execution for deferred statements)